### DISCUSSION TITLE
关于扩展被自动禁用

### DISCUSSION THREAD

**Discussion by whystars** [BewlyBewly/BewlyBewly#752] [Q&A]
我在使用这个扩展时，每当我更改完背景，bilibili就自动禁用了这个扩展，该如何解决？？？

**Comment by dbxsj** [comment]
使用chorm商店下载的bewlybewly然后导出设置，接着关闭，再打开edge扩展里的bewlybewly然后导入设置，亲测有效

**Comment by hakadao** [comment]
edge 瀏覽器太高端，本地測試和chrome上架都沒有這個問題edge偏偏遙遙領先過人
類似問題回答太多，我已經不想解釋，被edge餵太多屎了
# 建議用 chrome web store 中下載 BewlyBewly，其次不要用 edge 瀏覽器
